kubeluigi/k8s.py

```diff
@@ -271,23 +284,43 @@ def has_job_started(job: V1Job) -> bool:
                 if cond.reason == "Unschedulable":
                     if is_pod_waiting_for_scale_up(cond):
                         if has_scaling_failed(cond):
-                            logger.info(f"{logs_prefix} Cluster could not scale up.")
+                            logger.debug(f"{logs_prefix} Cluster could not scale up.")
                             raise FailedJob(
                                 job=job,
                                 message=f"Job: {job.metadata.name} - Pod: {pod.metadata.name} Failed to scale up : {cond.reason}  {cond.message}",
                             )
-                        logger.info(f"{logs_prefix} Waiting for cluster to Scale up..")
+                        logger.debug(f"{logs_prefix} Waiting for cluster to Scale up..")
                         return False
-    logger.info("Logs for the job can be found here: https://portal.azure.com/#blade/Microsoft_Azure_Monitoring/AzureMonitoringBrowseBlade/logs")
+    logger.debug(
+        "Logs for the job can be found here: https://portal.azure.com/#blade/Microsoft_Azure_Monitoring/AzureMonitoringBrowseBlade/logs"
+    )
     logs_queries = get_job_pod_logs_query(job)
     for query in logs_queries:
-        logger.info(f"Find logs for this jobs using the following query: {query}")
+        logs_query_link = get_query_link(query)
+        logger.info(f"Pod Logs: {logs_query_link}")
     return True
 
-def get_job_pod_logs_query(job:V1Job):
-    """ 
-        Returns a list of queries to use in azure monitor to retrieve logs for the pods running
-        for the scheduled job
+
+def get_query_link(query):
```

Comment:
we should bring this function to moussaka and let users inject their own implemetnations here in kubeluigi.

⚠️  tenant ids and subscriptions ids in a public repo..